<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewMessage</class>
 <widget class="QDialog" name="NewMessage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>926</width>
    <height>682</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>100</width>
    <height>75</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>New Message</string>
  </property>
  <widget class="QTextEdit" name="messageBody">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>250</y>
     <width>691</width>
     <height>341</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="sendButton">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>630</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Send</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>630</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QLabel" name="bodyLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>230</y>
     <width>46</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Body</string>
   </property>
  </widget>
  <widget class="QFrame" name="headerFrame">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>871</width>
     <height>201</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="toLabel">
      <property name="text">
       <string>To:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1" colspan="2">
     <widget class="QLineEdit" name="toLine"/>
    </item>
    <item row="0" column="3">
     <widget class="QLabel" name="ccLabel">
      <property name="text">
       <string>CC:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="4">
     <widget class="QLineEdit" name="ccLine"/>
    </item>
    <item row="0" column="5">
     <widget class="QLabel" name="bccLabel">
      <property name="text">
       <string>BCC:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="6">
     <widget class="QLineEdit" name="bccLine"/>
    </item>
    <item row="1" column="0" colspan="2">
     <widget class="QLabel" name="subjectLabel">
      <property name="text">
       <string>Subject:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2" colspan="3">
     <widget class="QLineEdit" name="subjectLine"/>
    </item>
    <item row="2" column="0" colspan="3">
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>Add Attachments</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>230</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Attachments:</string>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>250</y>
     <width>161</width>
     <height>341</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
